MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/model/MarvelImage.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient.model;
+
+import com.google.gson.annotations.SerializedName;
+
+public class MarvelImage {
+  @SerializedName("path") private String path;
+  @SerializedName("extension") private String extension;
+
+  public String getPath() {
+    return path;
+  }
+
+  public String getExtension() {
+    return extension;
+  }
```

Comment:
Can we add some methods to get the image in different sizes? [Here](http://developer.marvel.com/documentation/images) you have the documentation.